File: src/main/java/com/procurement/access/model/dto/lots/LotUpdateResponseDto.java
package com.procurement.access.model.dto.lots;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.procurement.access.model.dto.ocds.Lot;

public class LotUpdateResponseDto {

    @JsonProperty(value = "lot")
    private Lot lot;

    @JsonCreator
    public LotUpdateResponseDto(@JsonProperty("lot") final Lot lot) {
        this.lot = lot;
    }
}

File: src/main/kotlin/com/procurement/access/model/dto/lots/LotUpdateResponseDto.kt
package com.procurement.access.model.dto.lots

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude
import com.procurement.access.model.dto.ocds.Lot

@JsonInclude(JsonInclude.Include.NON_NULL)
data class LotUpdateResponseDto @JsonCreator constructor(

        val lot: Lot
)
